### PR TITLE
replacing for loop with tqdm for loop

### PR DESCRIPTION
**Pull request by rgreen1995**
replacing range with twdm.trange, this simple prints a progress bar

### CODE REVIEW

**Comment by hoyc1** [comment]
@rgreen1995, Can you please update this branch so the new unit tests are included.

**Comment by rgreen1995** [comment]
this contains some unrelated changes too ....

**Comment by rgreen1995** [comment]
addresses #15

File: PTMCMCSampler/PTMCMCSampler.py [hunk 1]
```diff
@@ -11,7 +11,8 @@
 from .nutsjump import NUTSJump, HMCJump, MALAJump
 from . import proposals as prop
 from .result import Result
-
+import tqdm
```
Comment (hoyc1) [inline PTMCMCSampler/PTMCMCSampler.py]: Is this line necessary?

**Pull request closed without merging**